src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx

```diff
@@ -245,15 +371,15 @@ If neither of the above statuses is suitable, assign the error to the appropriat
 
 ### Link errors to JIRA [#jira]
-As you get into more edge-case or complex errors, you might find you need to ask for help from other teams. Linking errors inbox to Jira may help with this. Connecting your errors inbox to Jira will enable you to easily create tickets connected to error groups. You can control the information sent to Jira via Jira templates.
+As you get into more edge-case or complex errors, you might find you need to ask for help from other teams. Linking errors inbox to Jira may help with this. [Connect your Errors inbox to Jira](/docs/errors-inbox/error-external-services#jira) to enable you to easily create tickets connected to error groups. You can control the information sent to Jira via Jira templates.
 
 ### Link errors to Slack [#slack]
-As the velocity of errors coming into rrors inbox goes down, a regular team session may no longer be a good use of time. An alternative is to link errors inbox to Slack and either a) designate a rotating person to monitor the channel and resolve/ignore/assign error groups as they come in or b) allow the team to proactively respond to error groups.
+As the velocity of errors coming into errors inbox goes down, a regular team session may no longer be a good use of time. An alternative is to [link Errors inbox to Slack](/docs/errors-inbox/error-external-services#slack) and either a) designate a rotating person to monitor the channel and resolve/ignore/assign error groups as they come in or b) allow the team to proactively respond to error groups.
 
 ### Use CodeStream [#codestream]
-Many of your error groups are going to require code changes to resolve. [Integrating CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) allows you to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa. 
+Many of your error groups are going to require code changes to resolve. [Integrate CodeStream with New Relic](/docs/codestream/start-here/what-is-codestream) to allow you to open the offending code directly in your IDE to investigate the code directly. You can also leave notes and comments on specific lines of code for developers to review and vice versa. 
 
-Integrating CodeStream with errors inbox also gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.
+The CodeStream integration gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.
```

Comment:
```suggestion
New Relic with CodeStream gives you more context on your error groups, such as being able to see version numbers or commit SHAs. Additionally, using errors inbox as a centralized place to identify, discuss, and rectify code issues allows you to respond to code problems efficiently and avoid duplicating work.
```

Reply:
I think replace "The CodeStream integration" with "New Relic with CodeStream" or even just "CodeStream" is the right call.